### DISCUSSION TITLE
Event with more than one day

### DISCUSSION THREAD

**Discussion by hko01** [liedekef/events-made-easy#161] [Q&A]
Hi,
is it possible to define an event which has more than one day ?

Example:
# English Session
# Takes place from 01th May to 29th May, Every Monday btw 09:00 and 11:00

If someone books this event, a calendar entry with 5 days btw 09:00 and 11:00 should be generated. So these are not single events, but one on 5 days....

BR Holger

**Comment by liedekef** [comment]
recurring events are possible

**Comment by hko01** [comment]
but recurring means that in my example 5 single events are created and the user must book each single event. 
I would need a multi-day event which occurs 5 mondays ...

**Comment by liedekef** [comment]
Sorry, but that is not possible. You can then create 1 event (in which they can subscribe) and a recurring for the other mondays ..
